src/components/SfacActivityNotification.js

```diff
@@ -0,0 +1,218 @@
+import React, { useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Button from '@material-ui/core/Button'
+import PropTypes from 'prop-types'
+import { Typography } from '@material-ui/core'
+import CheckCircleIcon from '@material-ui/icons/CheckCircle'
+import DoNotDisturbOnIcon from '@mui/icons-material/DoNotDisturbOn'
+import { GET_SEARCH_URL, SFAC_FEEDBACK_LINK } from 'src/utils/urls'
+import Link from 'src/components/Link'
+import Notification from './Notification'
+
+const useStyles = makeStyles((theme) => ({
+  noButton: {
+    height: '30px',
+    fontWeight: '500',
+    fontFamily: 'Poppins',
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(1),
+  },
+  noLink: {
+    paddingTop: theme.spacing(0.5),
+    height: '30px',
+    fontWeight: '500',
+    fontFamily: 'Poppins',
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(1),
+  },
+  yesLink: {
+    paddingTop: theme.spacing(0.5),
+    paddingLeft: theme.spacing(1.5),
+    paddingRight: theme.spacing(1.5),
+    background: '#29BEBA',
+    borderRadius: '15px',
+    height: '30px',
+    fontWeight: '900',
+    fontFamily: 'Poppins',
+    marginLeft: theme.spacing(1),
+    color: 'white',
+  },
+  title: {
+    fontWeight: '700',
+    fontSize: '24px',
+    fontFamily: 'Poppins',
+  },
+  text: {
+    paddingBottom: theme.spacing(3.5),
+  },
+  buttonsWrapper: {
+    display: 'flex',
+    justifyContent: 'flex-end',
+  },
+  active: {
+    color: '#219653',
+  },
+  inactive: {
+    color: '#E3720E',
+  },
+  status: {
+    display: 'flex',
+    flexDirection: 'row',
+    paddingBottom: theme.spacing(2.5),
+  },
+  statusWrapper: {
+    paddingBottom: theme.spacing(2.5),
+  },
+  searchCountContainer: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  searchCountSubContainer: {
+    flexGrow: 1,
+    paddingRight: theme.spacing(2),
+  },
+  statusText: {
+    fontFamily: 'Poppins',
+    paddingRight: theme.spacing(0.5),
+    fontWeight: 700,
+  },
+  searchCount: {
+    fontWeight: 900,
+  },
+  statusIcon: {
+    paddingTop: theme.spacing(0.5),
+    height: theme.spacing(2.5),
+  },
+}))
+const SfacExtensionSellNotification = ({
+  className,
+  activityState,
+  searchesToday,
+  totalSearches,
+  impactName,
+}) => {
+  const [open, setOpen] = useState(true)
+  const classes = useStyles()
+  const onNotNowClick = () => {
+    setOpen(false)
+  }
+  let buttons = null
+  if (activityState === 'inactive') {
+    buttons = (
+      <div className={classes.buttonsWrapper}>
+        <Link
+          to={SFAC_FEEDBACK_LINK}
+          className={classes.noLink}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          FEEDBACK
```

Comment:
Thanks for changing to links! However, these should also still use the Button component (consistent UI when interacting, likely better accessibility).